Tasks/AzureIoTEdgeV2/deployimage.ts

```diff
@@ -177,6 +211,37 @@ class azureclitask {
       throw resultOfToolExecution;
     }
   }
+
+  private static async getIdToken(connectedService: string) : Promise<string> {
```

Comment:
This looks pretty much similar to what we already have in [ApplicationTokenCredentials](https://github.com/microsoft/azure-pipelines-tasks/blob/cfce1ff7929bf1562faf4e54648da75c4be1dcec/common-npm-packages/azure-arm-rest-v2/azure-arm-common.ts#L444). Would it be possible to reuse it to prevent duplicating code?

Reply:
Good point, I will try to find some common package where I can put this.